nixos/modules/services/networking/firewall.nix

```diff
@@ -331,6 +331,16 @@ in
           '';
       };
 
+      package = mkOption {
+        type = types.package;
+        default = pkgs.iptables;
+        example = pkgs.iptables-compat;
```

Comment:
I think we can reconsider using `iptables-nft` by default if `nftables` is enabled.